SConstruct

```diff
@@ -60,8 +61,9 @@ envCython["CCFLAGS"] += ["-Wno-#warnings", "-Wno-shadow", "-Wno-deprecated-decla
 envCython["CCFLAGS"].remove("-Werror")
 
 python_libs = []
-if arch == "Darwin":
-  envCython["LINKFLAGS"] = ["-bundle", "-undefined", "dynamic_lookup"]
+# macOS (both Intel and Apple Silicon)
+if sys.platform == "darwin":
```

Comment:
is `arch` not equivalent?

Reply:
On Intel-based systems, it returns x86_64, while on Apple Silicone systems, it returns arm64. sys.platform returns darwin. envCython is the same for both, so I figured this was a nice quick way to do both without complicating logic.